src/src/ESP8266_WebUpdate.cpp

```diff
@@ -5,6 +5,9 @@
 #include <ESP8266WebServer.h>
 #include <ESP8266mDNS.h>
 #include <ESP8266HTTPUpdateServer.h>
+#include "SX127xDriver.h"
+
+extern SX127xDriver Radio;
```

Comment:
We need a better architecture for passing instances of objects around :(

Reply:
teach me >.<